### PR TITLE
updates to PPDM chapter

### PR DESCRIPTION
**Pull request by ladylazarus3**
Hi @elizhobart,  here are the updates to the PPDM chapter that brings the manufacturer instructions into alignment with the publisher and distributor instructions.

i also did a line by line comparison for all instructions over the various types of PPDM elements (Parallel name of X, Name of X, Date of X, etc.) and made everything a bit smoother.  Understandably, over the course of the last few months the text had shifted around a bit.  PPDM was always a bit of a guinea pig chapter as far as text transfer and Github development went, so it was good to have a second look through everything.

kalan

### CODE REVIEW

File: _pages/PPDM/Note-on-manufacture-statement.md [hunk 1]
```diff
@@ -22,40 +22,35 @@ sidebar:
...
 
-<a name="4.226.4.1">4.226.4.1</a> Make a note on manufacture statement if necessary to justify a supplied [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) (see [4.221.4.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.4.1)).
+<a name="4.226.4.1">4.226.4.1</a> Make a note if the manifestation is issued in more than one physical part, and the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) changes in the course of publication by giving the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) of the later part(s) (see [4.221.9.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.9.1)).
 
-<a name="4.226.4.2">4.226.4.2</a> If the place name as transcribed is known to be fictitious or incorrect, make a note on manufacture giving the actual place name. Include the source of the actual place name (see [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/), [4.221.5.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.5.1)).
+<a name="4.226.4.2">4.226.4.2</a> Make an explanatory note if the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) appearing on the manifestation is known to be fictitious or incorrect (see [4.221.96.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.96.1)).
```
Comment (elizhobart) [inline _pages/PPDM/Note-on-manufacture-statement.md:39]: Should be "Make a note on manufacture statement giving the basis for the correction if the ..." (continue from [Place of manufacture] unchanged) to mirror instructions at 4.206.4.2.

File: _pages/Ppdm/Parallel-place-of-manufacture.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ sidebar:
...
 
-<a name="4.212.3.2">4.212.3.2</a> Transcribe remaining [Places of manufacture](/DCRMR/ppdm/Place-of-manufacture/) as parallel places of manufacture. If a parallel place of manufacture is from a different source than the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/), make a [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/) if considered important for identification (see [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/), [4.226.7.1](/DCRMR/ppdm/Note-on-manufacture-statement/#4.226.7.1)).
+<a name="4.222.3.2">4.222.3.2</a> Transcribe remaining [Places of manufacture](/DCRMR/ppdm/Place-of-manufacture/) as parallel places of manufacture. If a parallel place of manufacture is from a different source than the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/), make a [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/) if considered important for identification (see [Note on manufacture statement](/DCRMR/ppdm/Note-on-manufacture-statement/), [4.226.7.1](/DCRMR/ppdm/Note-on-manufacture-statement/#4.226.7.1)).
```
Comment (elizhobart) [inline _pages/Ppdm/Parallel-place-of-manufacture.md:28]: Transcribe remaining Places of Publication" not hyperlinked in Parallel place of publication. This should be consistent.

**Review by elizhobart — CHANGES_REQUESTED** [review]

File: _pages/PPDM/Name-of-publisher.md [hunk 3]
```diff
@@ -185,13 +187,13 @@ If a statement such as “Privately printed” appears on the title page, transc
...
+<a name="4.203.96.1">4.203.96.1</a> If the name of the distributor does not appear in the [Publisher statement](/DCRMR/ppdm/Publisher-statement/), and the name of the publisher has already been transcribed as part of another element, supply it within square brackets.
 
 >Example:  
 >Name of publisher: <CITE>[Pour Symo[n] Vostre Libraire par Philippe Pigouchet]</CITE>  
 >(*Comment*: Title reads “Ces presentes heures a lusaige de Ro[m]me : so[n]t au lo[n]g sa[n]s req[ue]rir [et] ont este faictes pour Symo[n] Vostre Libraire: demourent a Paris a la rue neuue nostre dame, a lenseigne Sainct Iehan leuangeliste par Philippe Pigouchet.”)
 
-If transcribing a publisher name as part of another element, do not abridge or expand the Name of publisher simply because it repeats or omits information given elsewhere in the description.
+If transcribing a publisher name as part of another element, do not abridge or expand the name of publisher simply because it repeats or omits information given elsewhere in the description.
 
 ## 4.203.97 Publisher names in more than one language or script
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Name-of-publisher.md:199]: updated language to match parallel name of distributor and parallel name of manufacturer

File: _pages/PPDM/Name-of-publisher.md [hunk 4]
```diff
@@ -185,13 +187,13 @@ If a statement such as “Privately printed” appears on the title page, transc
 
 ## 4.203.96 Publisher names transcribed as part of another element 
 
-<a name="4.203.96.1">4.203.96.1</a> If no formal name of publisher appears on the manifestation, and the name of the publisher has already been transcribed as part of another element, supply it within square brackets.
+<a name="4.203.96.1">4.203.96.1</a> If the name of the distributor does not appear in the [Publisher statement](/DCRMR/ppdm/Publisher-statement/), and the name of the publisher has already been transcribed as part of another element, supply it within square brackets.
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Name-of-publisher.md]: fixed.  Thanks!

File: _pages/PPDM/Name-of-publisher.md [hunk 5]
```diff
@@ -156,19 +158,19 @@ If a statement such as “Privately printed” appears on the title page, transc
 >Example:  
 ><CITE>Carter and Hendee [and 20 others in 18 places]</CITE>
 
-<a name="4.203.7.3">4.203.7.3</a> If the name of a publisher associated with an earlier edition appears together with the name of the actual publisher of the edition being described, transcribe the names as a single element in the order in which they appear.
+<a name="4.203.7.3">4.203.7.3</a> If both the place and publisher associated with an earlier edition appears together with the name of the actual publisher of the edition being described, transcribe the names as a single element in the order in which they appear.
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Name-of-publisher.md:162]: fixed!  Thanks! :)

File: _pages/PPDM/Name-of-publisher.md [hunk 6]
```diff
@@ -78,7 +78,7 @@ If a statement such as “Privately printed” appears on the title page, transc
...
 
-<a name="4.203.5.1">4.203.5.1</a> If only the address, sign, or initials of the publisher appear in lieu of the name, transcribe the statement containing the address, sign, or initials as the [Publication statement](/DCRMR/ppdm/Publication-statement/). If the name of publisher can be identified, supply it in square brackets after the initials or before or after the address or sign, as appropriate, or give the information in a [Note on publication statement](/DCRMR/ppdm/Note-on-publication-statement/) (see [4.206.5.1](/DCRMR/ppdm/Note-on-publication-statement/#4.206.5.1)).
+<a name="4.203.5.1">4.203.5.1</a> If only the address, sign, or initials of the distributor, appears in lieu of the name, transcribe the statement containing the address, sign, or initials as the name of distributor. If the publisher's name can be identified, supply it in square brackets after the initials or before or after the address or sign, as appropriate, or give the information in a [Note on publication statement](/DCRMR/ppdm/Note-on-publication-statement/) (see [4.206.5.1](/DCRMR/ppdm/Note-on-publication-statement/#4.206.5.1)).
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Name-of-publisher.md]: fixed!  Thanks!

File: _pages/PPDM/Name-of-publisher.md [hunk 7]
```diff
@@ -78,7 +78,7 @@ If a statement such as “Privately printed” appears on the title page, transc
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Name-of-publisher.md:78]: fixed!  Thanks!

File: _pages/PPDM/Name-of-distributor.md [hunk 8]
```diff
@@ -62,21 +62,25 @@ sidebar:
...
+<a name="4.213.6.1">4.213.6.1</a> If the name of distributor is known to be fictitious or incorrect, transcribe it nonetheless and make an explanatory note. If the actual details are known, or can be reasonably surmised, supply the correction and its basis in a [Note on distribution statement](/DCRMR/ppdm/Note-on-distribution-statement/) (see [4.216.5.3](/DCRMR/ppdm/Note-on-distribution-statement/#4.216.5.3)).
+
+<a name="4.213.6.2">4.213.6.2</a> **Alternative rule**. If the name as transcribed is known to be fictitious, transcribe it as it appears and supply in square brackets the actual name, if known. Give the basis for the correction or clarification in a [Note on distribution statement](/DCRMR/ppdm/Note-on-distribution-statement/) (see [4.216.5.3](/DCRMR/ppdm/Note-on-distribution-statement/#4.216.5.3)).
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Name-of-distributor.md]: thanks!  fixed.

File: _pages/PPDM/Date-of-publication.md [hunk 9]
```diff
@@ -87,7 +87,7 @@ sidebar:
 
 <a name="4.205.4.4">4.205.4.4</a> **Very long dates**. If the date on the preferred source of information is very long, generally transcribe it as it appears.
 
-<a name="4.205.4.5">4.205.4.5</a> *Optionally,* Substitute for it a formalized statement in square brackets. If the supplied date includes a day/month, use the sequence: day, month, year. Make a [Note on publication statement](/DCRMR/ppdm/Note-on-publication-statement/) concerning the source and the original form of the date (see [4.206.8.2](/DCRMR/ppdm/Note-on-publication-statement/#4.206.8.2)).
+<a name="4.205.4.5">4.205.4.5</a> *Optionally,* substitute for it a formalized statement in square brackets. If the supplied date includes a day/month, use the sequence: day, month, year. Make a [Note on publication statement](/DCRMR/ppdm/Note-on-publication-statement/) concerning the source and the original form of the date (see [4.206.8.2](/DCRMR/ppdm/Note-on-publication-statement/#4.206.8.2)).
```
[the remark this answers is not in the document]
Comment (ladylazarus3) [reply _pages/PPDM/Date-of-publication.md]: brought three elements into alignment for this instruction.

**Pull request merged**